res/ui/Tactile/flow.js

```diff
@@ -427,4 +424,4 @@ function run(loader) {
     initialise_steps(loader);
     create_steps();
     run_steps();
-}
+}
```

Comment:
We end our files with a [newline](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file). You can change your editor by following [this guide](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files).

Reply:
thank you a lot Tom!